### DISCUSSION TITLE
Sunbird-ED/Cokreat Regression Sign off for R5.1.0

### DISCUSSION THREAD

**Discussion by ShailandraOjha** [Sunbird-Ed/Community#352] [General Discussion]
Hi All,

This is the sign-off for R5.1.0 for Sunbird-ED/Cokreat as We have completed the new feature verification, CName migration and Regression testing on the Sunbird-ED environment, Please find the detailed test case execution sheet [Here](https://docs.google.com/spreadsheets/d/1wUb5kSMpPFfY7KQ3NaFLX84zRQEr1-wm6N8Z-N8i-ZI/edit#gid=1981725419)

Environment: Sunbird-ED
Testing completion percentage: 100% Completed
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10614)

Note: Following open defect will be taken in UAT. 

         1-https://project-sunbird.atlassian.net/browse/KN-731

![image](https://user-images.githubusercontent.com/44188853/212723933-50d6f905-e933-40fe-9af5-a6f11172f82f.png)

​Also please find the Regression + New feature sign-off sheets from all the circles [here](https://docs.google.com/spreadsheets/d/1lF2IqugS_-iJNfxrrkc-7tTmk6gfzLIl6ZuFGb-CbWw/edit#gid=762862810).

![image](https://user-images.githubusercontent.com/44188853/212725990-e568f946-5000-4476-a295-e6507416e98f.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai @madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @swayangjit @tarang27 @anunagaraj

Regards,
Shailandra Ojha

**Comment by anisha147** [comment]
Hi Shailandra

Release notes 5.1.0 is now available on the microsite.

Thanks and regards,
Anisha

**Comment by Krishnaj20** [comment]
> Hi All,
> 
> This is the sign-off for R5.0 for Sunbird-ED/Cokreat as We have completed the new feature verification, CName migration and Regression testing on the Sunbird-ED environment, Please find the detailed test case execution sheet [Here](https://docs.google.com/spreadsheets/d/1wUb5kSMpPFfY7KQ3NaFLX84zRQEr1-wm6N8Z-N8i-ZI/edit#gid=1981725419)
> 
> Environment: Sunbird-ED Testing completion percentage: 100% Completed Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10614)
> 
> Note: Following open defect will be taken in UAT.
> 
> ```
>      1-https://project-sunbird.atlassian.net/browse/KN-731
> ```
> 
> ![image](https://user-images.githubusercontent.com/44188853/212723933-50d6f905-e933-40fe-9af5-a6f11172f82f.png)
> 
> ​Also please find the Regression + New feature sign-off sheets from all the circles [here](https://docs.google.com/spreadsheets/d/1lF2IqugS_-iJNfxrrkc-7tTmk6gfzLIl6ZuFGb-CbWw/edit#gid=762862810).
> 
> ![image](https://user-images.githubusercontent.com/44188853/212725990-e568f946-5000-4476-a295-e6507416e98f.png)
> 
> @rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai @madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @swayangjit @tarang27 @anunagaraj
> 
> Regards, Shailandra Ojha

@ShailandraOjha - This should be 5.1.0 release right? You've mentioned 5.0

**Reply by ShailandraOjha** [reply]
@Krishnaj20 Updated
